packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -32,16 +37,26 @@ function BlockGroupToolbar( { label = _x( 'Group', 'verb' ) } ) {
 		[ clientIds ]
 	);
 
-	const onConvertToGroup = () => {
+	const onConvertToGroup = ( variation = 'group' ) => {
 		const newBlocks = switchToBlockType(
 			blocksSelection,
 			groupingBlockName
 		);
-		if ( newBlocks ) {
+
+		if ( newBlocks && newBlocks.length > 0 ) {
+			if ( variation !== 'group' ) {
```

Comment:
We can probably avoid this special case by just putting "undefined" as a value in the object above.